pkgs/applications/editors/vim/plugins/updater.nix

```diff
@@ -0,0 +1,53 @@
+{ buildPythonApplication
+, nix
+, makeWrapper
+, python3Packages
+, lib
+
+# optional
+, updateTreesitter ? true
+, vimPlugins
+, neovim
+}:
+let
+  my_neovim = neovim.override {
+    configure.packages.all.start = [ vimPlugins.nvim-treesitter ];
+  };
+
+in
+buildPythonApplication {
+  format = "other";
+  pname = "vim-plugins-updater";
+  version = "0.1";
+
+  nativeBuildInputs = [
+    makeWrapper
+    python3Packages.wrapPython
+  ];
+
+  pythonPath = [
+    python3Packages.gitpython
+  ];
+
+  dontUnpack = true;
+
+  env = lib.optionals updateTreesitter  { NVIM_TREESITTER = "${vimPlugins.nvim-treesitter}/lockfile.json"; };
```

Comment:
I don't think that would work, since that package would build before the vim plugins are updated, and this nvim-treesitter would be from before the update.

Maybe we don't want this, and we should just wait for the nixpkgs CLI which would possibly make running maintainer scripts easier

Reply:
haha yeah makes sense

> Maybe we don't want this, and we should just wait for the nixpkgs CLI which would possibly make running maintainer scripts easier

not sure what you are referencing to. Any link ?